std/assembly/builtins.ts

```diff
@@ -226,6 +226,10 @@ export namespace atomic {
   // @ts-ignore: decorator
   @builtin
   export declare function fence(): void;
+
+  export function isLockFree(size: usize): bool {
+    return size == 1 || size == 2 || size == 4;
```

Comment:
What about `8` in context of Wasm?

Reply:
See jfbastien's [comment](https://github.com/WebAssembly/design/issues/300#issue-101454763)

Reply:
I think 8-byte lock free could be guarantee only for `wasm64` platform